Files/App.xaml.cs

```diff
@@ -446,6 +446,42 @@ private void OnSuspending(object sender, SuspendingEventArgs e)
 
         private static void AppUnhandledException(Exception ex)
         {
+#if DEBUG
+            Debug.WriteLine("--------- UNHANDLED EXCEPTION ---------");
+            if (ex != null)
+            {
+                Debug.WriteLine($"\n>>>> HRESULT: {ex.HResult}\n");
+                if (ex.Message != null)
+                {
+                    Debug.WriteLine("\n--- MESSAGE ---");
+                    Debug.WriteLine(ex.Message);
+                }
+                if (ex.StackTrace != null)
+                {
+                    Debug.WriteLine("\n--- STACKTRACE ---");
+                    Debug.WriteLine(ex.StackTrace);
+                }
+                if (ex.Source != null)
+                {
+                    Debug.WriteLine("\n--- SOURCE ---");
+                    Debug.WriteLine(ex.Source);
+                }
+                if (ex.InnerException != null)
+                {
+                    Debug.WriteLine("\n--- INNER ---");
+                    Debug.WriteLine(ex.InnerException);
+                }
+            }
+            else
+            {
+                Debug.WriteLine("\nException is null!\n");
+            }
+
+            Debug.WriteLine("---------------------------------------");
+
+            Debugger.Break(); // Please check "Output" Window for exception details (View -> Output Window) (CTRL + ALT + O)
+#endif
```

Comment:
Can this be done in a separate pull request?

Reply:
Is it necessary?

Reply:
Yes, it's something we are going to be enforcing more in order to better track changes.

Reply:
Ok